code/modules/surgery/organs/external/spines.dm

```diff
@@ -14,36 +14,27 @@
 
 	bodypart_overlay = /datum/bodypart_overlay/mutant/spines
 
-	///A two-way reference between the tail and the spines because of wagging sprites. Bruh.
-	var/obj/item/organ/external/tail/lizard/paired_tail
-
 /obj/item/organ/external/spines/Insert(mob/living/carbon/receiver, special, movement_flags)
-	. = ..()
-	if(.)
-		paired_tail = locate(/obj/item/organ/external/tail/lizard) in receiver.organs //We want specifically a lizard tail, so we don't use the slot.
-		paired_tail?.paired_spines = src
+	// If we have a tail, attempt to add a tail spines overlay
+	var/obj/item/organ/external/tail/our_tail = receiver.get_organ_slot(ORGAN_SLOT_EXTERNAL_TAIL)
+	our_tail?.try_insert_tail_spines(our_tail.bodypart_owner)
```

Comment:
```suggestion
	our_tail?.try_insert_tail_spines(receiver)
```

Reply:
bodypart_owner here is a bodypart, rather than the receiver _mob_. try_insert_tail_spines() takes a bodypart as input since it adds a bodypart overlay.

Reply:
Oops